### PR TITLE
Fix various line-height styles

### LINKED ISSUE #26537: Strange green point in dashboard repo list
### Description

I think this comes from #26520
image
(Using Chrome on Windows)
@silverwind 

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by wxiaoguang**
Fix #26537

Use the same default line-height as "normalize.css" instead of "1". "1" is not right because it doesn't work with descent part and causes overflow problems.

image

### CODE REVIEW

**Comment by silverwind** [comment]
Maybe `normal` is also a good default, but I see this 1.15 even survived into `modern-normalize`.

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height
https://github.com/sindresorhus/modern-normalize/blob/c997a1e1d37aadbf9b6bc46ed396e7d9ebc8632f/modern-normalize.css#L29

**Review by delvh — APPROVED** [review]
More CSS variables?
Yes please.
(Optimally, in the future we would only use variables and nothing else, but that's just a dream of me…)

File: web_src/css/repo.css [hunk 1]
```diff
@@ -390,7 +390,7 @@
 }
 
 .repository.file.list .non-diff-file-content .header .file-actions .btn-octicon {
-  line-height: 1;
+  line-height: var(--line-height-default);
   padding: 10px 8px;
```
Comment (silverwind) [inline web_src/css/repo.css]: De we need to adjust this to something like `9px 8px` to maintain previous element height?

File: web_src/css/base.css [hunk 2]
```diff
@@ -518,6 +520,7 @@ a.label,
 .ui.menu .item {
   color: var(--color-text);
   user-select: auto;
+  line-height: var(--line-height-default); /* fomantic uses "1" which causes overflow problems because "1" doesn't consider the descent part */
 }
```
Comment (silverwind) [inline web_src/css/base.css:526]: Also here, maybe we should adjust vertical padding to counteract the line-height increase that made the element larger in height.

Comment (silverwind) [reply web_src/css/base.css:526]: Fixed in https://github.com/go-gitea/gitea/pull/26553/commits/9ba7096ba456dbf9910ee6477a3ca018308d57a8.

File: web_src/css/repo.css (hunk 1 above)
Comment (silverwind) [reply web_src/css/repo.css]: Fixed in https://github.com/go-gitea/gitea/pull/26553/commits/9ba7096ba456dbf9910ee6477a3ca018308d57a8.

**Review by silverwind — APPROVED** [review]

**Pull request merged**